src/PactNet.Native/NativeMessagePactBuilder.cs

```diff
@@ -120,5 +114,31 @@ public async Task VerifyAsync<T>(Func<T, Task> handler)
         }
 
         #endregion Internal Methods
+
+        #region Private Methods
+
+        /// <summary>
+        /// Try to read the reified message
+        /// </summary>
+        /// <typeparam name="T">the type of message</typeparam>
+        /// <returns>the message</returns>
+        private T MessageReified<T>()
+        {
+            string reified = this.server.Reify(this.message);
+            NativeMessage content = JsonConvert.DeserializeObject<NativeMessage>(reified);
```

Comment:
Yeah this is using Newtonsoft but the other side appears to use System.Text.Json. I imagine that'll cause all kinds of wird incompatibility and unexpected outcomes.

Reply:
As I said in my previous message, we started to ask for both ^^ Let's agree to use only newtonsoft then.

Reply:
Newtonsoft approach pushed